### ISSUE TITLE
Firefox compatibility 

### ISSUE THREAD

**Issue by darshancc23** [darshancc23/pay-to-pdf#4]
The app only works on chrome. Need a fix to make it work on other browsers especially Firefox.

To try out the app: 

- Clone the repository
- Run the index file on chrome
- Fill the form
- Click on Pay with Razorpay
- Pay using any method. (for card please use 4111-1111-1111-1111)

The PDF should download within a few seconds.

**Comment by lonesta** [comment]
I try.

**Comment by lonesta** [comment]
I try you app in Firefox (62.0.3) and Safari (12.0 (14606.1.36.1.9))

No mistakes, everything works as it should.

**Comment by darshancc23** [comment]
Are you sure the PDF is downloaded on a successful transaction?

**Comment by GustavoMantuan** [comment]
May I take a look?

**Comment by GustavoMantuan** [comment]
I'll take a look then.

**Comment by darshancc23** [comment]
@lonesta Yes it somehow seems to work now, @GustavoMantuan Did you try it as well?

**Comment by GustavoMantuan** [comment]
I'm taking a look right now. @darshancc23

**Comment by GustavoMantuan** [comment]
Firefox 59 - 64bits 
not working here! I'll take a look.

**Comment by darshancc23** [comment]
@GustavoMantuan Thank you

**Comment by GustavoMantuan** [comment]
@darshancc23 Hey, maybe I have found the problem,
as we are not using SPA, when we use: window.location right after the pdf save, sometimes for firefox the file are not ready yet and we are "refreshing" the browser, so the file doesn't show up for download.

The best solution here (in my opinion) are to make a button for user to navigate back to index.html, and not forcing by window.location.
So by removing the code window.location='./index.html' it should fix the problem for all browsers, because we give enough time to "download" the file.

I made a PR if you want to check it out.

**Comment by darshancc23** [comment]
@GustavoMantuan The reason I forcefully redirected the user to the from was to simply improve the user experience. The user wouldn't want to click a button to go back to the form page and there's also a chance that the user will simply hit the back button on the browser which breaks the app.

**Comment by darshancc23** [comment]
@GustavoMantuan However, If we don't have a choice we can implement the button.

**Comment by GustavoMantuan** [comment]
@darshancc23 Hmm, in that case, we could use a delay (setTimeout) and then use window.location but it will "freeze" the app for a couple seconds.

**Comment by darshancc23** [comment]
@GustavoMantuan I see, send a PR and I'll check it out.

**Comment by GustavoMantuan** [comment]
@darshancc23 I think we have a problem with the timeout fix, we could put 5s timeout, but, for some reason (network, problem with the api, browser) the time could no be enough to generate the file, so we still facing a problem here.
